xfce/orage/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem 1.0
 
 name            orage
-version         4.12.1
+version         4.18.0
 set branch      [join [lrange [split ${version} .] 0 1] .]
 categories      xfce
 platforms       darwin
```

Comment:
"platforms darwin" is now the default so it can be removed.

Reply:
Fixed.